plots/measure.js

```diff
@@ -119,47 +147,117 @@ const URLS = [
   'https://bbc.com'
 ];
 
-/**
- * Launches Chrome once at the beginning, runs all the analysis,
- * and then kills Chrome.
- * TODO(chenwilliam): measure the overhead of starting chrome, if it's minimal
- * then open a fresh Chrome instance for each run.
- */
+function getUrls() {
+  if (args['sites-path']) {
```

Comment:
since we already have this, why not toss the default SITES into a file and have sites-path default to that one.